Core/ContentBlockerConfigurationUserDefaults.swift

```diff
@@ -66,10 +72,15 @@ public class ContentBlockerProtectionUserDefaults: ContentBlockerProtectionStore
     public func isTempUnprotected(domain: String?) -> Bool {
         guard let domain = domain else { return false }
         
+        if tempUnprotectedDomains == nil {
+            tempUnprotectedDomains = FileStore().loadAsArray(forConfiguration: .temporaryUnprotectedSites)
+                .filter { !$0.trimWhitespace().isEmpty }
+        }
+        
         // Break domain apart to handle www.*
         var tempDomain = domain
         while tempDomain.contains(".") {
-            if tempUnprotectedDomains.contains(tempDomain) {
+            if tempUnprotectedDomains!.contains(tempDomain) {
```

Comment:
😬 

don’t like this force unwrap, would be better to keep it optional and either default to false using ?? false or check explicitly against the bool using == true